admin/src/scenes/classe/view.tsx

```diff
@@ -0,0 +1,311 @@
+import React, { useState, useEffect } from "react";
+import { HiOutlineOfficeBuilding } from "react-icons/hi";
+import { AiOutlinePlus } from "react-icons/ai";
+import { BsSend, BsTrash3 } from "react-icons/bs";
+import { useParams, useHistory } from "react-router-dom";
+import { useSelector } from "react-redux";
+import dayjs from "dayjs";
+import { toastr } from "react-redux-toastr";
+
+import { Page, Header, Button, Badge, DropdownButton } from "@snu/ds/admin";
+import { capture } from "@/sentry";
+import api from "@/services/api";
+import { translate, ROLES, YOUNG_STATUS, STATUS_CLASSE, translateStatusClasse, COHORT_TYPE, IS_INSCRIPTION_OPEN_CLE, FUNCTIONAL_ERRORS } from "snu-lib";
+import { getRights, statusClassForBadge } from "./utils";
+import { appURL } from "@/config";
+import Loader from "@/components/Loader";
+import plausibleEvent from "@/services/plausible";
+import { downloadCertificatesByClassId } from "@/services/convocation.service";
+import { usePendingAction } from "@/hooks/usePendingAction";
+import { Classe } from "@/types";
+import { AuthState } from "@/redux/auth/reducer";
+import { CohortState } from "@/redux/cohorts/reducer";
+
+import GeneralInfos from "./components/GeneralInfos";
+import ReferentInfos from "./components/ReferentInfos";
+import SejourInfos from "./components/SejourInfos";
+import StatsInfos from "./components/StatsInfos";
+import DeleteButton from "./components/DeleteButton";
+import ModaleWithdraw from "./components/modale/ModaleWithdraw";
+import ModaleInvite from "./components/modale/ModaleInvite";
+import { InfoBus, TStatus, Rights } from "./components/types";
+
+export default function View() {
+  const [classe, setClasse] = useState<Classe | null>(null);
+  const [url, setUrl] = useState("");
+  const [studentStatus, setStudentStatus] = useState<{ [key: string]: number }>({});
+  const [modaleInvite, setModaleInvite] = useState(false);
+  const [modaleWithdraw, setModaleWithdraw] = useState(false);
```

Comment:
```suggestion
  const [showModaleWithdraw, setShowModaleWithdraw] = useState(false);
```
ce n'est pas forcement le sujet de cette PR vu que c'est du split mais il serait bien que le nom du state nous indique ce qu'il fait. pour les `boolean` on prefixe par `is`, `has` ou `show`